src/components/FileItem/FileItem.tsx

```diff
@@ -1,17 +1,21 @@
-import React, { useEffect, useState } from "react";
+import { useEffect, useState } from "react";
 import { useNavigate, useParams } from "react-router-dom";
 import {
   useChangeFileMutation,
   useDeleteFileMutation,
   useGetFileByNameQuery,
 } from "../../services/task1";
 import { Box, Button, Card, Grid, TextField, Typography } from "@mui/material";
+import { File } from "../../types";
 
 export const FileItem = () => {
   const { filename } = useParams();
   const navigate = useNavigate();
 
-  const { data, error, isLoading } = useGetFileByNameQuery(filename!);
+  const { data } = filename
+    ? useGetFileByNameQuery(filename)
```

Comment:
Well eslint error is self descriptive but I actually meant this
https://redux-toolkit.js.org/rtk-query/usage/conditional-fetching